From 4aa04407d5622216496f472eedb8cf2f851b589a Mon Sep 17 00:00:00 2001
From: Amos Kong <akong@redhat.com>
Date: Wed, 15 Jan 2014 09:58:05 +0100
Subject: [PATCH 36/39] rng-egd: remove redundant free

RH-Author: Amos Kong <akong@redhat.com>
Message-id: <1389779886-26583-3-git-send-email-akong@redhat.com>
Patchwork-id: 56725
O-Subject: [RHEL-6.6 qemu-kvm PATCH 2/3] rng-egd: remove redundant free
Bugzilla: 786407
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>

We didn't set default chr_name, the free is redundant.

Signed-off-by: Amos Kong <akong@redhat.com>
Message-id: 1385023371-8198-2-git-send-email-akong@redhat.com
Signed-off-by: Anthony Liguori <aliguori@amazon.com>
(cherry picked from commit 5e490b6a504912225dff0e520e1c6af68295d238)
---
 backends/rng-egd.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 backends/rng-egd.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/backends/rng-egd.c b/backends/rng-egd.c
index ad84737..d653641 100644
--- a/backends/rng-egd.c
+++ b/backends/rng-egd.c
@@ -162,7 +162,6 @@ static void rng_egd_set_chardev(Object *obj, const char *value, Error **errp)
     if (b->opened) {
         error_set(errp, QERR_PERMISSION_DENIED);
     } else {
-        g_free(s->chr_name);
         s->chr_name = g_strdup(value);
     }
 }
-- 
1.7.1

